spec/helpers/application_helper/buttons/view_spec.rb

```diff
@@ -31,11 +31,11 @@
     before { subject.calculate_properties }
 
     context 'when gtl_type is set' do
-      it_behaves_like 'a disabled button'
+      include_examples 'ApplicationHelper::Button::Basic#disabled?', nil
```

Comment:
@skateman [`ApplicationHelper::Button::View#disabled?`](https://github.com/ManageIQ/manageiq-ui-classic/blob/7769b5cad7cb8623009de2e6e5573cf68a92f590/app/helpers/application_helper/button/view.rb#L7-L9) does not set a title message on disabled. Should it be considered as bad implementation, or may the test be slightly altered to accept error messages that are `nil`?